runtime-modules/council/src/lib.rs

```diff
@@ -723,21 +735,35 @@ impl<T: Trait> EnsureChecks<T> {
         let account_id = ensure_signed(origin)?;
 
         ensure!(
-            T::CandidacyLock::is_member_staking_account(&council_user_id, &account_id),
+            T::is_council_member_account(council_user_id, &account_id),
             Error::CouncilUserIdNotMatchAccount,
         );
 
         Ok(account_id)
     }
 
-    /// Checks there are no problems with existing candidacy record (if any present).
-    fn ensure_previous_stake_reusable(
+    fn ensure_user_membership_staking(
```

Comment:
This method does two unrelated things, and the name gives ambiguous semantics. Please split out check for membership signer on origin from check that given member controls a staking account.

Reply:
Missing response.

Reply:
Done